kong/pdk/tracing.lua

```diff
@@ -37,15 +33,18 @@ local ngx_ERR = ngx.ERR
 
 local NOOP = function() end
 
-local FLAG_SAMPLED = 0x01
-local FLAG_RECORDING = 0x02
-local FLAG_SAMPLED_AND_RECORDING = bit.bor(FLAG_SAMPLED, FLAG_RECORDING)
-
 local POOL_SPAN = "KONG_SPAN"
 local POOL_SPAN_STORAGE = "KONG_SPAN_STORAGE"
 local POOL_ATTRIBUTES = "KONG_SPAN_ATTRIBUTES"
 local POOL_EVENTS = "KONG_SPAN_EVENTS"
 
+-- must be power of 2
+local SAMPLING_BYTE = 8
+local SAMPLING_PRECISION = 8 * SAMPLING_BYTE
```

Comment:
since `SAMPLING_BYTE` is the number of byte**s** we consider from the trace id to apply the sampling algorithm on, should we rename to the plural `SAMPLING_BYTES`?

Reply:
Guess "bits" is more precise.

Reply:
oh yeah SAMPLING_BITS instead of SAMPLING PRECISION, but I was referring to what is now `SAMPLING_BYTE = 8` which refers to the number of bytes on this line: https://github.com/Kong/kong/pull/10485/files#diff-ee1d8aa5ddb77d9d9724f6bd21f6da4b6f4ed563a4cdaf0d53d5b7a40ca93652R99